images/ubuntu-bun-ai/Dockerfile

```
@@ -10,17 +10,20 @@ ARG USERNAME=root
 # Elevate to root to install personal taste additions
 USER root
 
-# 1. Install CodeRabbit CLI
+# 1. Install system cron
+RUN apt-get update && apt-get install -y --no-install-recommends cron && rm -rf /var/lib/apt/lists/*
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)